modules/core/keeper/msg_server.go

```diff
@@ -190,7 +190,7 @@ func (k Keeper) ChannelOpenInit(goCtx context.Context, msg *channeltypes.MsgChan
 	// Perform application logic callback
 	version, err := cbs.OnChanOpenInit(ctx, msg.Channel.Ordering, msg.Channel.ConnectionHops, msg.PortId, channelID, cap, msg.Channel.Counterparty, msg.Channel.Version)
 	if err != nil {
-		return nil, sdkerrors.Wrap(err, "channel open init callback failed")
+		return nil, sdkerrors.Wrapf(err, "channel open init callback failed for Port ID: %s, Channel ID: %s", msg.PortId, channelID)
```

Comment:
```suggestion
		return nil, sdkerrors.Wrapf(err, "channel open init callback failed for port ID: %s, channel ID: %s", msg.PortId, channelID)
```

I think most of our errors avoid capitals so I think it'd make sense to be consistent

Reply:
That's great! will convert these to lower case